nms/packages/magmalte/app/views/equipment/EnodebDetailConfigEdit.js

```diff
@@ -472,6 +579,329 @@ export function RanEdit(props: Props) {
+                  placeholder="Enter LTEINTRAA5THRESHOLD1RSRP"
+                  fullWidth={true}
+                  value={optConfig.lte_intra_a5_threshold_1_rsrp}
+                  onChange={({target}) =>
+                    handleOptChange(
+                      'lte_intra_a5_threshold_1_rsrp',
+                      target.value,
+                    )
+                  }
+                />
+              </AltFormField>
+              <AltFormField label={'lte intraa5 threshold2 rsrp'}>
+                <OutlinedInput
+                  data-testid="lte_intra_a5_threshold_2_rsrp"
+                  placeholder="Enter LTEINTRAA5THRESHOLD2RSRP"
+                  fullWidth={true}
+                  value={optConfig.lte_intra_a5_threshold_2_rsrp}
+                  onChange={({target}) =>
+                    handleOptChange(
+                      'lte_intra_a5_threshold_2_rsrp',
+                      target.value,
+                    )
+                  }
+                />
+              </AltFormField>
+              <AltFormField label={'lte interanra5 threshold1 rsrp'}>
+                <OutlinedInput
+                  data-testid="lte_inter_anr_a5_threshold_1_rsrp"
+                  placeholder="Enter LTEINTERANRA5THRESHOLD1RSRP"
+                  fullWidth={true}
+                  value={optConfig.lte_inter_anr_a5_threshold_1_rsrp}
+                  onChange={({target}) =>
+                    handleOptChange(
+                      'lte_inter_anr_a5_threshold_1_rsrp',
+                      target.value,
+                    )
+                }
+                />
+              </AltFormField>
+              <AltFormField label={'lte interanra5 threshold2 rsrp'}>
+                <OutlinedInput
+                  data-testid="lte_inter_anr_a5_threshold_2_rsrp"
+                  placeholder="Enter LTEINTERANRA5THRESHOLD2RSRP"
+                  fullWidth={true}
+                  value={optConfig.lte_inter_anr_a5_threshold_2_rsrp}
+                  onChange={({target}) =>
+                    handleOptChange(
+                      'lte_inter_anr_a5_threshold_2_rsrp',
+                      target.value,
+                    )
+                  }
+                />
+              </AltFormField>
+              <AltFormField label={'b2 threshold1 rsrp'}>
+                <OutlinedInput
+                  data-testid="b2_threshold1_rsrp"
+                  placeholder="Enter B2THRESHOLD1RSRP"
+                  fullWidth={true}
+                  value={optConfig.b2_threshold1_rsrp}
+                  onChange={({target}) =>
+                    handleOptChange('b2_threshold1_rsrp', target.value)
+                  }
+                />
+              </AltFormField>
+              <AltFormField label={'b2 threshold2 rsrp'}>
+                <OutlinedInput
+                  data-testid="b2_threshold2_rsrp"
+                  placeholder="Enter B2THRESHOLD2RSRP"
```

Comment:
Is there a particular reason for the all-caps placeholders? I don't imagine that this is particularly legible.

Reply:
![image](https://user-images.githubusercontent.com/87301024/147404018-06bb91c6-d190-4ac3-9ee6-ce54abe25505.png)
Modified, thank you!